packages/space-opera/src/components/ibl_selector/ibl_selector.ts

```diff
@@ -86,6 +87,15 @@ export class IblSelector extends ConnectedLitElement {
             this.config.environmentImage) {
       reduxStore.dispatch(dispatchEnvrionmentImage(
           dropdownElement.selectedItem.getAttribute('value') || undefined));
+      // dropdown value equals null when "Default" is selected
+      if (dropdownElement.selectedItem.getAttribute('value') === null) {
+        reduxStore.dispatch(dispatchSetEnvironmentName(undefined));
```

Comment:
Not sure where this error is, but everything works great when you choose one of our supplied models and environments. However, if I load my own, the zip no longer contains a GLB at all, and the HDR comes down as "undefined", though it is the correct file.

Reply:
fixed